Makefile

```diff
@@ -138,7 +137,6 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 .PHONY: generate-operator-yaml
 generate-operator-yaml: manifests
 	$(KUSTOMIZE) build deploy/manager > deploy/operator.yaml
-
```

Comment:
Not sure why this blank line was removed?

Reply:
https://github.com/stolostron/config-policy-controller/pull/531#discussion_r1258371546 Matt asked me to do this.

Reply:
I think he didn't mean to take out the line--just the extra tab whitespace 🙂

Reply:
OH!! Thanks! you save me